### DISCUSSION TITLE
Symbols don't display properly in IntelliJ IDEA

### DISCUSSION THREAD

**Discussion by wonktondI** [starship/starship#3823] [Q&A] [answered]
windows10, I install starship with cargo.
I use git-bash as my terminal.
When I use it in a different IDE:
IDEA:
![image](https://user-images.githubusercontent.com/41692176/161104022-ad1323b0-30e7-465e-b0bb-84ea46035c4e.png)
VS Code
![image](https://user-images.githubusercontent.com/41692176/161103976-ef1c526c-a263-455a-a835-4d26b8870d3a.png)

🙏help me
thanx :)

**Comment by matchai** [comment, marked as answer]
It looks like this is a tracked upstream issue in IntelliJ IDEA: https://youtrack.jetbrains.com/issue/IDEA-248010
I'd recommend giving the issue a "thumbs up".

**Reply by wonktondI** [reply]
😥

**Reply by davidkna** [reply]
Alternatively, you could avoid emoji and use the [nerd font preset](https://starship.rs/presets/nerd-font.html).
